modules/_canvas-kit-react/README.md

```diff
@@ -1,3 +1,8 @@
-# Canvas Kit
+# Canvas Kit React
 
 The bundle package containing all modules of the Canvas React Kit.
+
+> Note: By default, no fonts are included with Canvas Kit modules. To use official Canvas Kit fonts,
+> install and import the `@workday/canvas-kit-react-fonts` module in addition to
+> `@workday/canvas-kit-react`. The fonts files are served from the Workday CDN so this allows
```

Comment:
I feel like the language here is odd since you probably are a consumer looking at this. Maybe something like:

> By installing `@workday/canvas-kit-react-fonts` you are opting in to using the Workday CDN.